src/Glpi/Inventory/Asset/VirtualMachine.php

```diff
@@ -341,9 +357,29 @@ protected function createVmComputer()
                     if ($this->conf->states_id_default != '-1') {
                         $input['states_id'] = $this->conf->states_id_default;
                     }
-                    $datarules = $rule->processAllRules($input);
+                    $datarules = $rule->processAllRules($input, [], ['class' => $input['itemtype'], 'return' => true]);
```

Comment:
```suggestion
                    $datarules = $rule->processAllRules($input, [], ['class' => $input['itemtype']]);
```

`'return' => true` in `$params` does not seem to exist or to be treated by the rules

Reply:
I guess the entire param is useless; `class` parameter shoudl only be a `InventoryAsset` instance. See existing usages and `RuleImportAsset` class.

Reply:
I would suggest not changing this part (the PR doesn't need it).

Reply:
Hello guys,
I think it's needed since without it, when running the ``processAllRules``, it automatically creates a log into "Import information" tab into the asset. We create the log manually since it contains more information than the one provided by the default log facility, as the GLPI-Agent that triggered the update. This is also the behavior of GLPI 11.